en/add_wagtail_to_your_website/wagtail_integration_adding_posts/README.md

```diff
@@ -0,0 +1,130 @@
+    ]
+
+    content_panels = Page.content_panels + [
+        FieldPanel('date'),
+        FieldPanel('intro'),
+        FieldPanel('text'),
+    ]
+```
+
+Before we move on, we will make some comparisons between our `Post` and `BlogPage` models. 
+- Our `Post` model contains a field named but we can't use the same field name in our `BlogPage` model because our base 
+model `Page` already contains a field by that name and hence this will cause an error. 
+- Our `Post` model also has fields`author` and `published_date` which we omitted in our `BlogPage` model because our 
+base model `Page` has a field `owner`that represents the `author` and has a `-first_published_at` field that represents 
+the `published_date`. 
+- We also do not have the `publish` method as this is again handled by a method in our `Page` model. 
+- The `Page` class also takes care of the string representation for our `BlogPage` model hence we do not have to define 
+a `__str__()` method in the model as we already inherited that. 
+- Wagtail also takes care of the search functionality for our `BlogPage` model. For this we only need to define 
+`search_fields`, something we would need to do in `admin.py` for our `Post` model. We imported `index` in `BlogPage` 
+because it makes our model searchable and then we listed the fields we wanted to use for searching the model.
+
+As you can see, Wagtail takes care of most of the work we would have done for ourselves by writing our own code to 
+have that functionality. Among Wagtail's many powerful capabilities is the search capability for which Wagtail provides 
+two backends for storing the search index and performing search queries: one using the database's full-text search 
+capabilities, and another using Elasticsearch. You can also roll your own search backend. Isn't that awesome?
+
+We can now run `python3 manage.py makemigrations`command to make migrations for `BlogPage`
+
+```bash
+(myvenv) ~/djangogirls$ python3 manage.py makemigrations 
+``` 
+
+and the migrate command to create the `BlogPage` model as shown below:
+
+```bash
+(myvenv) ~/djangogirls$ python3 manage.py migrate
+```
+
+Now that we have created our `BlogPage` model, we need to add a template `blog_page.html` in our `blog/templates/blog` 
+folder and add the following:
+
+{% filename %} blog_page.html {% endfilename %}
+```html
+{% extends "wagtail_base.html" %}
+
+{% load wagtailcore_tags %}
+
+{% block body_class %}template-blogpage{% endblock %}
+
+{% block content %}
+    <h1>{{ page.title }}</h1>
+    <p class="meta">{{ page.date }}</p>
+
+    <div class="intro">{{ page.intro }}</div>
+
+    {{ page.body|richtext }}
+
+    <p><a href="{{ page.get_parent.url }}">Return to blog</a></p>
+
+{% endblock %}
+```
+
+Wagtail’s built-in `get_parent()` method  is used to obtain the URL of the blog this post is a part of.
+
+Now we can create a few blog posts as children of `Home Page`. To do this, navigate to `Pages` and click on the arrow
```

Comment:
Wagtail has pretty good documentation of the CMS interface if you wanted to refer people to it: https://guide.wagtail.org/en-latest/

Though it’s not available in that many languages currently, compared to Django Girls tutorials.

Reply:
Will add it.